pig/src/main/resources/interpreter-setting.json

```diff
@@ -15,6 +15,12 @@
         "propertyName": "zeppelin.pig.includeJobStats",
         "defaultValue": "false",
         "description": "flag to include job stats in output"
+      },
+      "zeppelin.pig.execType": {
+        "envName": "SPARK_MASTER",
+        "propertyName": "SPARK_MASTER",
```

Comment:
wait, this doesn't seem quite right - `SPARK_MASTER` is not the same as `zeppelin.pig.execType`?
if someone sets `zeppelin.pig.execType` to `tez` and `SPARK_MASTER` to `local`?